packages/core/package.json

```diff
@@ -32,7 +32,7 @@
     "lint": "eslint -c .eslintrc.js --ext .ts .",
     "prepare": "yarn build",
     "report": "Rscript bin/report.R test/results/results.json",
-    "test": "yarn lint && yarn mocha",
+    "test": "yarn lint && yarn mocha \"test/test.ts\" \"test/trace/*.ts\"",
```

Comment:
Is this something you want in the final PR?

Reply:
yes, otherwise only the top-level tests get run. But I want to organize the tests in folders from now on.